examples/gallery/embellishments/timestamp.py

```diff
@@ -24,12 +23,14 @@
 # label font can be defined via the ``font`` parameter and the timestamp string
 # format via ``timefmt``.
```

Comment:
Strictly speaking the font set via the `font` parameter impacts on both the timestamp string and the addional label.

```suggestion
# font can be defined via the ``font`` parameter and the timestamp string
# format via ``timefmt``.
```